stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -292,7 +292,7 @@ public fun MessageComposer(
             mentionPopupContent(mentionSuggestions)
         }
 
```

Comment:
Could you remind me what corner case is covered by the second part of this condition (`messageComposerState.attachments.isEmpty()`)? Editing a message with attachments?

Reply:
If you don't use this, any time `commandSuggestions` contains suggestions the command suggestions popup will appear.

This means that if you have attachemnts present and type in '/' inside the `MessageComposer` input without checking if any attachments are present the commad suggestion list popup will appear.

We want to avoid this because sending commands along with attachments is confusing UX and results in us effectively suggesting users they can do both at the same when that is not true. Unfortunately they can still manually type in a command but it's better if we don't actively suggest it.